plugins/mcreator-core/procedures/gui_get_value_checkbox.json

```diff
@@ -6,10 +6,9 @@
       "text": "checkBoxName"
     },
     {
-      "type": "field_image",
```

Comment:
Keep client indicators for this PR, as this PR alone will not make this server-side